pkg/manager/member/tiflash_member_manager_test.go

```diff
@@ -642,7 +643,7 @@ func TestTiFlashMemberManagerSyncTidbClusterStatus(t *testing.T) {
 		}
 	}
 	tests := []testcase{
-		{
+		/*{
```

Comment:
Why comment this out?

Reply:
Sorry I was trying to debug test case and forgot to recover it.